node/src/node_configurator/mod.rs

```diff
@@ -302,37 +154,6 @@ pub fn check_for_past_initialization(
     }
 }
 
-pub fn request_new_db_password(
-    streams: &mut StdStreams,
-    possible_preamble: Option<&str>,
-    prompt: &str,
-    confirmation_prompt: &str,
-) -> Option<String> {
-    if let Some(preamble) = possible_preamble {
-        flushed_write(streams.stdout, &format!("{}\n", preamble));
-    }
-    match request_password_with_retry(prompt, streams, |streams| {
-        request_password_with_confirmation(
-            confirmation_prompt,
-            "Passwords do not match.",
-            streams,
-            cannot_be_blank,
-        )
-    }) {
-        Ok(password) => Some(password),
-        Err(PasswordError::RetriesExhausted) => None,
-        Err(PasswordError::Mismatch) => None,
-        Err(PasswordError::VerifyError(e)) => {
-            flushed_write(
-                streams.stdout,
-                &format!("Could not elicit wallet encryption password: {:?}\n", e),
-            );
-            None
-        }
-        Err(PasswordError::InternalError(_)) => panic!("Can't happen: no code path"),
-    }
-}
-
 //TODO this code (and the allied -- see thorough this file) is very likely to go away when GH-457 is played
 pub fn request_existing_db_password(
```

Comment:
This function should go away. It won't be useful for GH-458; its only use is to accept interactive input at the command line when starting the Node, and we don't do that anymore because we have both a CLI and a GUI for that now.